timemachine/cpp/src/tibd_exchange_move.cu

```diff
@@ -74,7 +79,7 @@ TIBDExchangeMove<RealType>::TIBDExchangeMove(
     d_temp_storage_buffer_.realloc(temp_storage_bytes_);
 
     // Set the offset to the length of the random vector to ensure noise is triggered on first step
-    this->noise_offset_ = this->d_uniform_noise_buffer_.length;
+    this->noise_offset_ = this->RANDOM_BATCH_SIZE;
```

Comment:
same consideration as for the BD Exchange Move (does this have to be a member?)

Reply:
Removed in [4b11f08](https://github.com/proteneer/timemachine/pull/1234/commits/4b11f08431034c911b5998f8c441240252ba5b59) after offline discussion